promotion/replication/replication_iskhakov_et_al_2016.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<p>Very nice. Just a little typo: "contractions teps implemented". There is a "s" wrong :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/ruspy/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>